package.json

```diff
@@ -25,6 +25,8 @@
     "detox-test": "detox test --configuration ios.sim.debug"
   },
   "dependencies": {
+    "@actions/core": "^1.2.6",
```

Comment:
Is it possible for these to be `devDependecies`?

Reply:
I _think_ so. Do you know of a good way to know for sure?

Reply:
I thought it was if it's not used in the source, then it's not needed, but I am not sure off the top of my head which is installed when.

Reply:
Yeah, I think these can be dev dependencies.

Reply:
good call